Unable to mark an MkRelease with prerelease=true

Encountered this issue while writing tests in a separate project.

`Release.Smart#prerelease(boolean)` is not tested. I added the following in `MkReleaseTest` and notice that it fails as well:

```java
    @Test
    public void prerelease() throws Exception {
        final Release release = MkReleaseTest.release();
        final Release.Smart smart = new Release.Smart(release);
        smart.prerelease(true);
        MatcherAssert.assertThat(
            smart.prerelease(),
            Matchers.is(true)
        );
    }
```

Error is:

```
java.lang.AssertionError: 
Expected: is <true>
     but: was <false>
```

> @rultor release it, tag is `0.39`

@amihaiemil @llorllale Oops, I failed. You can see the full log [here](http://www.rultor.com/t/14434-386534259) (spent 8min)

```
               
Downloaded: https://repo.maven.apache.org/maven2/org/sonatype/sisu/sisu-inject-bean/2.5.3/sisu-inject-bean-2.5.3.jar (0 B at 0.0 KB/sec)
               
Downloaded: https://repo.maven.apache.org/maven2/org/sonatype/sisu/sisu-guice/3.1.6/sisu-guice-3.1.6-no_aop.jar (0 B at 0.0 KB/sec)
[INFO] LICENSE found: file:/home/r/repo/LICENSE.txt
log4j:WARN No appenders could be found for logger (com.puppycrawl.tools.checkstyle.ConfigurationLoader).
log4j:WARN Please initialize the log4j system properly.
log4j:WARN See http://logging.apache.org/log4j/1.2/faq.html#noconfig for more info.
[ERROR] /src/test/java/com/jcabi/github/RtUserTest.java[481]: First sentence in a comment should start with a capital letter (RegexpMultilineCheck)
[ERROR] /src/test/java/com/jcabi/github/RtUserTest.java[492]: Name 'markAsReadOkIfResponseStatusIs205' must match pattern '^[a-z]{2,}[a-zA-Z]+$'. (MethodNameCheck)
[ERROR] /src/test/java/com/jcabi/github/RtUserTest.java[499]: First sentence in a comment should start with a capital letter (RegexpMultilineCheck)
[ERROR] /src/test/java/com/jcabi/github/RtUserTest.java[506]: Name 'markAsReadErrorIfResponseStatusIsNot205' must match pattern '^[a-z]{2,}[a-zA-Z]+$'. (MethodNameCheck)
[ERROR] /src/main/java/com/jcabi/github/RtUser.java[149]: The String "notifications" appears 2 times in the file. (MultipleStringLiteralsCheck)
[INFO] Read our quality policy: http://www.qulice.com/quality.html
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 06:06 min
[INFO] Finished at: 2018-05-04T08:27:20+00:00
[INFO] Final Memory: 54M/462M
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal com.qulice:qulice-maven-plugin:0.13:check (jcabi-check) on project jcabi-github: Failure: 5 Checkstyle violations (see log above) -> [Help 1]
org.apache.maven.lifecycle.LifecycleExecutionException: Failed to execute goal com.qulice:qulice-maven-plugin:0.13:check (jcabi-check) on project jcabi-github: Failure
	at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:212)
	at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:153)
	at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:145)
	at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject(LifecycleModuleBuilder.java:116)
	at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject(LifecycleModuleBuilder.java:80)
	at org.apache.maven.lifecycle.internal.builder.singlethreaded.SingleThreadedBuilder.build(SingleThreadedBuilder.java:51)
	at org.apache.maven.lifecycle.internal.LifecycleStarter.execute(LifecycleStarter.java:128)
	at org.apache.maven.DefaultMaven.doExecute(DefaultMaven.java:307)
	at org.apache.maven.DefaultMaven.doExecute(DefaultMaven.java:193)
	at org.apache.maven.DefaultMaven.execute(DefaultMaven.java:106)
	at org.apache.maven.cli.MavenCli.execute(MavenCli.java:863)
	at org.apache.maven.cli.MavenCli.doMain(MavenCli.java:288)
	at org.apache.maven.cli.MavenCli.main(MavenCli.java:199)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced(Launcher.java:289)
	at org.codehaus.plexus.classworlds.launcher.Launcher.launch(Launcher.java:229)
	at org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode(Launcher.java:415)
	at org.codehaus.plexus.classworlds.launcher.Launcher.main(Launcher.java:356)
Caused by: org.apache.maven.plugin.MojoFailureException: Failure
	at com.qulice.maven.CheckMojo.doExecute(CheckMojo.java:65)
	at com.qulice.maven.AbstractQuliceMojo.execute(AbstractQuliceMojo.java:175)
	at org.apache.maven.plugin.DefaultBuildPluginManager.executeMojo(DefaultBuildPluginManager.java:134)
	at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:207)
	... 20 more
Caused by: com.qulice.spi.ValidationException: 5 Checkstyle violations (see log above)
	at com.qulice.checkstyle.CheckstyleValidator.validate(CheckstyleValidator.java:96)
	at com.qulice.maven.CheckMojo.run(CheckMojo.java:83)
	at com.qulice.maven.CheckMojo.doExecute(CheckMojo.java:59)
	... 23 more
[ERROR] 
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoFailureException
container 10cb20d9d465cceadd18518e5a2329a990296f7216bfca5844e7e26fe83b12d6 is dead
Fri May  4 10:27:27 CEST 2018
```

--- a/pom.xml
+++ b/pom.xml
@@ -261,7 +261,7 @@
     <dependency>
       <groupId>com.jcabi</groupId>
       <artifactId>jcabi-github</artifactId>
-      <version>0.38</version>
+      <version>0.39</version>
     </dependency>
     <dependency>
       <groupId>org.cactoos</groupId>

--- a/src/test/java/org/llorllale/mvn/plgn/releasecat/UploadTest.java
+++ b/src/test/java/org/llorllale/mvn/plgn/releasecat/UploadTest.java
@@ -16,11 +16,12 @@
 
 package org.llorllale.mvn.plgn.releasecat;
 
-// @checkstyle AvoidStaticImport (4 lines)
+// @checkstyle AvoidStaticImport (5 lines)
 import static org.hamcrest.CoreMatchers.is;
 import static org.junit.Assert.assertFalse;
 import static org.junit.Assert.assertNotNull;
 import static org.junit.Assert.assertThat;
+import static org.junit.Assert.assertTrue;
 
 import com.jcabi.github.Release;
 import com.jcabi.github.ReleaseAsset;
@@ -152,17 +153,13 @@ public final class UploadTest {
    * 
    * @throws Exception unexpected
    * @since 0.2.0
-   * @todo #4:15min jcabi-github has a bug
-   *  (https://github.com/jcabi/jcabi-github/issues/1363) that makes it impossible to mark
-   *  a MkRelease as prerelease. When that is fixed, come back here and change 'assertFalse'
-   *  for 'assertTrue' and make sure it works.
    */
   @Test
   public void prepreleaseTrue() throws Exception {
     final Repo repo = new MkGithub("my_user").repos()
       .create(new Repos.RepoCreate("my_project", false));
     new Upload("Tag v1.0", "Name v1.0", "", true, () -> repo).execute();
-    assertFalse(
+    assertTrue(
       new Release.Smart(new Releases.Smart(repo.releases()).find("Tag v1.0")).prerelease()
     );
   }

